client/ionic/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@capacitor/android": "^1.5.1",
     "@capacitor/core": "1.5.1",
+    "@capacitor/ios": "^1.5.1",
```

Comment:
Our semantic versioning should be consistent.  Either all are caret compatible, or all are single version (for all three capacitor pkgs).

Reply:
updated in ea9ea80